moveit_core/kinematic_constraints/src/utils.cpp

```diff
@@ -273,3 +273,156 @@ moveit_msgs::Constraints kinematic_constraints::constructGoalConstraints(const s
 
   return goal;
 }
+
+bool kinematic_constraints::validatePositionConstraints(const robot_state::RobotState& state,
+                                                        moveit_msgs::Constraints& c)
+{
+  ROS_INFO("Validating position constraints.");
```

Comment:
add NAMED to these INFOs

Reply:
My bad, this file is the only one where I left my actual debugging lines. Will update after ROSCon JP.

Reply:
When no one is looking, I even use "print()" because it's quicker to type than "rospy.loginfo()".